CountItems/masterProcess.js

```diff
@@ -18,6 +22,14 @@ const loggerConfig = {
 werelogs.configure(loggerConfig);
 const log = new werelogs.Logger('S3Utils::CountItems::Master');
 
+const prometheusPollingFrequency = process.env.PROMETHEUS_POLLING_FREQUENCY && !Number.isNaN(process.env.PROMETHEUS_POLLING_FREQUENCY)
+    ? Number.parseInt(process.env.PROMETHEUS_POLLING_FREQUENCY, 10)
```

Comment:
should we not try to `parseInt()`, and default if it fails, so we never end up with an invalid prometheusPollingFrequency?
otherwise we need to assert/abort/fail if the variable is not valid (e.g. prometheusPollingFrequency is a number greater than 0).

Reply:
done here : https://github.com/scality/s3utils/pull/312/commits/7a546979810051418f9af9bc3ed2567a0bc69eb4#diff-46cbf734fd7b9022c7bbf597a8a5deb96829b581ca817a861bbc3bf1e86274a9R27-R29